### PR TITLE
R4R: The TokenAddition proposal 's InitialSupply should not be set

### PR DESCRIPTION
**Pull request by zhiqiang-bianjie**
Resolves: #2017

### CODE REVIEW

File: app/v1/gov/msgs.go [hunk 1]
```diff
@@ -361,7 +359,7 @@ func (msg MsgSubmitTokenAdditionProposal) ValidateBasic() sdk.Error {
 		return err
 	}
 
-	issueToken := exported.NewMsgIssueToken(exported.FUNGIBLE, exported.EXTERNAL, "", msg.Symbol, msg.CanonicalSymbol, msg.Name, msg.Decimal, msg.MinUnitAlias, msg.InitialSupply, exported.MaximumAssetMaxSupply, false, nil)
+	issueToken := exported.NewMsgIssueToken(exported.FUNGIBLE, exported.EXTERNAL, "", msg.Symbol, msg.CanonicalSymbol, msg.Name, msg.Decimal, msg.MinUnitAlias, 0, exported.MaximumAssetMaxSupply, false, nil)
```
Comment (zhangyelong) [inline app/v1/gov/msgs.go]: Mintable must be `true` here, otherwise the external token can never be minted

File: app/v1/gov/proposal_util.go [hunk 2]
```diff
@@ -101,10 +101,9 @@ func createTokenAdditionInfo() pTypeInfo {
 			return buildProposal(content, func(p BasicProposal, content Content) Proposal {
 				addTokenMsg := content.(MsgSubmitTokenAdditionProposal)
 				decimal := int(addTokenMsg.Decimal)
-				initialSupply := sdk.NewIntWithDecimal(int64(addTokenMsg.InitialSupply), decimal)
 				maxSupply := sdk.NewIntWithDecimal(int64(exported.MaximumAssetMaxSupply), decimal)
 
-				fToken := exported.NewFungibleToken(exported.EXTERNAL, "", addTokenMsg.Symbol, addTokenMsg.Name, addTokenMsg.Decimal, addTokenMsg.CanonicalSymbol, addTokenMsg.MinUnitAlias, initialSupply, maxSupply, false, nil)
+				fToken := exported.NewFungibleToken(exported.EXTERNAL, "", addTokenMsg.Symbol, addTokenMsg.Name, addTokenMsg.Decimal, addTokenMsg.CanonicalSymbol, addTokenMsg.MinUnitAlias, sdk.ZeroInt(), maxSupply, false, nil)
```
Comment (zhangyelong) [inline app/v1/gov/proposal_util.go]: Mintable must be `true` here, otherwise the external token can never be minted

**Comment by zhangyelong** [comment]
🎉 Approved, no need to update the docs, I'm doing the refactor.

**Comment by chengwenxi** [comment]
There may be some conflicts.

**Pull request merged**